src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -3109,4 +3109,7 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="StoreInstall_PackageNotAvailableForCurrentSystem" xml:space="preserve">
     <value>The package is not compatible with the current Windows version or platform.</value>
   </data>
+  <data name="ConfigurationSuppressPrologueArgumentDescription" xml:space="preserve">
+    <value>Suppress the configuration prologue when possible</value>

```

Comment:
I don't think this will be clear to either users or translators. Do we say "prologue" anywhere else?